src/api/minimal/iuf.rs

```diff
@@ -60,7 +60,7 @@ macro_rules! impl_minimal_iuf {
             /// If `index >= Self::lanes()` the behavior is undefined.
             #[inline]
             pub unsafe fn extract_unchecked(self, index: usize) -> $elem_ty {
-                use llvm::simd_extract;
+                use crate::llvm::simd_extract;
```

Comment:
the same applies to `crate::` here, if the `llvm` module end ups in `core::packed_simd::llvm` all of these imports would break.